spider/__main__.py

```diff
@@ -1,36 +1,39 @@
-# Vaccine image scraper
-# Copyright (C) 2020  Nguyễn Gia Phong
-#
-# This file is part of Spider Venom.
-#
-# Spider Venom is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as published
-# by the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Spider Venom is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with Spider Venom.  If not, see <https://www.gnu.org/licenses/>.
-
-from sys import argv
-
-from httpx import AsyncClient
-from trio import Path, open_nursery, run
-
-from .tuoitre import tuoitre
-from .vnexpress import vnexpress
-
-
-async def main(dest):
-    """Download vaccine images."""
-    async with AsyncClient() as client, open_nursery() as nursery:
-        nursery.start_soon(tuoitre, Path(dest), client, nursery)
-        nursery.start_soon(vnexpress, Path(dest), client, nursery)
-
-
-if __name__ == '__main__':
-    run(main, argv[1] if len(argv) > 1 else '.')
+# Vaccine image scraper
+# Copyright (C) 2020  Nguyễn Gia Phong
+#
+# This file is part of Spider Venom.
+#
+# Spider Venom is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Spider Venom is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Spider Venom.  If not, see <https://www.gnu.org/licenses/>.
+
+from sys import argv
+
+from httpx import AsyncClient
+from trio import Path, open_nursery, run
+
+
+from tuoitre import tuoitre
+from vnexpress import vnexpress
+from dantri import dantri
+
+
+async def main(dest):
+    """Download vaccine images."""
+    async with AsyncClient() as client, open_nursery() as nursery:
+        nursery.start_soon(tuoitre, Path(dest), client, nursery)
+        nursery.start_soon(vnexpress, Path(dest), client, nursery)
+        nursery.start_soon(dantri, Path(dest), client, nursery)
+
+
+if __name__ == '__main__':
+    run(main, argv[1] if len(argv) > 1 else '.')
```

Comment:
Ehm, what's up with such a hugh diff?

Reply:
oh ok, i got that, when i checked that in my computer, i should remove the dot in "from .dantri import dantri" and change by "from dantri import dantri" and the same with vnexpress and tuoitre then it worked for our code

Reply:
cd into the project's and `python -m spider`, what's so difficult with that

Reply:
in window 10, I can't use `python -m spider`, as you saw yesterday, i should have move the file into the cmd then i can run the code

Reply:
You can, @pdhlong is using Windows too.  Maybe you have to replace python with py but you have to figure it out.

Reply:
sorry, i can work with that :))  and everything work fine 
![abc](https://user-images.githubusercontent.com/59214933/100721379-d6465300-33f1-11eb-9567-9d2acfcfe198.PNG)

Reply:
> Also please check you source, the only scraped image is the portrait of the antitrust offender.